packages/suite-desktop/src-electron/app.ts

```diff
@@ -161,6 +163,21 @@ const init = async () => {
     // repeated during app lifecycle (e.g. Ctrl+R)
     ipcMain.handle('handshake/load-modules', (_, payload) => loadModulesResponse(payload));
 
+    // Tor module initializes separated from general `initModules` because Tor is different
+    // since it is allowed to fail and then the user decides whether to `try again` or `disable`.
+    const loadTorModule = initTorModule({
+        mainWindow,
+        store,
+        interceptor,
+    });
+
+    // create handler for handshake/load-tor-module
+    const loadTorModulesResponse = () => {
```

Comment:
This is kind of residue of general module architecture, which could be simplified even more, if we want to.
- by removing the `Module` typing of Tor module (which kind of makes sense as it isn't common module anymore) we could omit the `null` parameter of `loadTorModule(null)`
- also, it would then be possible to use `loadTorModule` directly instead of `loadTorModulesResponse` as it seems that `initTorModule` always returns value and therefore `throw new Error(...)` shouldn't happen

Reply:
Yes, I changed it and simplified. For some reason I wanted to keep the Module type even though it was not making sense anymore, so now it is way simpler.